package.json

```diff
@@ -18,7 +18,7 @@
     "build:dev": "tsc --project es2017.tsconfig.json",
     "format": "prettier --write \"src/**/*.ts\"",
     "test": "yarn test:node && yarn test:browser",
-    "test:node": "yarn proto && yarn plasmaabi && tsc && tape dist/tests/unit_tests.js | tap-spec",
+    "test:node": "mkdir -p db && yarn proto && yarn plasmaabi && tsc && tape dist/tests/unit_tests.js | tap-spec && rm -rf db",
```

Comment:
Clean this up in the tests, can use `shelljs` to do this kind of thing like so https://github.com/loomnetwork/loom-js/blob/master/scripts/plasma-abi.js#L5